### PR TITLE
changed the visuals and added a button

### PR DESCRIPTION
**Pull request by DemirBalemir**
**Background**
We will have multiple components on the web page. This PR is for their styling. 

**Changes**
I made the mentioned changes to the style.css file and created a new branch. The Google HTML-CSS guidelines are used for this file. https://google.github.io/styleguide/htmlcssguide.html#CSS

**Screenshot**

261264715-b6d30e81-6eb8-4b3d-919b-c3bf490c9112

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+    *{
```
Comment (nturgut) [inline style.css]: Remove the extra lines before the css style. Also put a space between the style name and the curly brackets.
In summary each style will look like this:
```
<StyleName> {
...
}
<empty line>
<StyleName2> {
...
}
```

File: style.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    margin : 0;
+    padding : 0;
+    box-sizing : border-box;
+    font-family : 'Poppins', sans-serif;
+}
+.hero{
+    background:#1d2026;
```
Comment (nturgut) [inline style.css]: use a space after the property name's colon.

Here is the Google style guide for explanation: https://google.github.io/styleguide/htmlcssguide.html#Property_Name_Stops

File: style.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    width: 100%;
+    color:#fff;
+    position: relative;
+}
+.text-control{
+    position: absolute;
+    top: 10%;
+    left: 30%;
+    font-size: 20px;
+
```
Comment (nturgut) [inline style.css]: remove the extra lines.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    color: #40E0D0;
+    font-size: 30px;
+}
+.button{
+   background-color: #1d2026;
+   border: 2px solid #1d2026;
+   color: #fff;
+   padding: 10px 20px;
+   text-align: center;
+}
```
Comment (nturgut) [inline style.css]: Going forward add a new empty line in the end of each file.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+.text-control{
+    position: absolute;
+    top: 10%;
+    left: 30%;
+    font-size: 20px;
+ 
+    
+    
+}
+.header{
```
Comment (nturgut) [inline style.css]: Good job, the css style names in the entire file are very well chosen!

**Review by nturgut — COMMENTED** [review]
I added a few comments.

You can use Google CSS Style in the future when working on new `css`: https://google.github.io/styleguide/htmlcssguide.html#CSS Feel free to add a link to README as well.

**Comment by nturgut** [comment]
> _ No description provided. _

I suggest setting up PR descriptions in this way:

**Background**
<Some information on why are we doing this code change>

**Change**
<Summarize what is changed in this PR>

**Screenshot**
This is good for frontend related PRs especially. Makes it easier to review styling changes.

**Review by nturgut — COMMENTED** [review]
I noticed that the PR description says: `changed the visuals and added a button #2`

I saw the `css` changes but I didn't see the button component. Is there an uncommitted file?

**Comment by DemirBalemir** [comment]
Website Screenshot

File: style.css [hunk 6]
```diff
@@ -0,0 +1,37 @@
+* {
```
Comment (nturgut) [inline style.css:1]: styling looks good! Feel free to mark the earlier comments as `Resolved`

**Review by nturgut — APPROVED** [review]
Looks good to me! (LGTM)

**Pull request merged**